src/gc/proxy.d

```diff
@@ -76,12 +76,30 @@ extern (C)
             }
             _instance = newInstance;
             // Transfer all ranges and roots to the real GC.
-            (cast(ProtoGC) protoInstance).term();
+            (cast(ProtoGC) protoInstance).transfer();
```

Comment:
Renaming seems reasonable, but the name could be even more descriptive making the comment obsolete, e.g. `transferRangesAndRoots`.